pkgs/development/libraries/capstone/capstone4.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, stdenv
+, cmake
+, fetchFromGitHub
+, fixDarwinDylibNames
+}:
+
+stdenv.mkDerivation rec {
+  pname = "capstone";
+  version = "4.0.2";
+
+  src = fetchFromGitHub {
+    owner = "aquynh";
```

Comment:
you should probably change owner here to capstone-engine since it's a redirect now